tmail-backend/tmail-third-party/ai-bot/src/test/java/com/linagora/tmail/mailet/AIBotMailetTest.java

```diff
@@ -53,18 +57,16 @@ public static MailAddress createMailAddress(String mailAddress) {
     private static final MailAddress BOT_ADDRESS = createMailAddress("gpt@localhost");
 
     private MailetContext mailetContext;
-    private Configuration configuration;
     private AIBotConfig aiBotConfig;
     private AIBotMailet testee;
 
     @BeforeEach
-    void setUp() {
-        configuration = new PropertiesConfiguration();
-        configuration.addProperty("apiKey", "demo");
-        configuration.addProperty("botAddress", "gpt@localhost");
-        configuration.addProperty("model", "gemini-2.0-flash");
-        configuration.addProperty("baseURL", "https://generativelanguage.googleapis.exemple.com");
-        aiBotConfig= AIBotConfig.fromAiPropertiesConfig(configuration);
+    void setUp() throws MalformedURLException, AddressException {
```

Comment:
```suggestion
    void setUp() throws Exception {
```

In tests we do not care about detailing exceptions